src/sage/combinat/designs/designs_pyx.pyx

```diff
@@ -15,6 +15,195 @@ from cysignals.memory cimport sig_malloc, sig_calloc, sig_realloc, sig_free
 
 from sage.misc.unknown import Unknown
 
+def is_covering_array(array,strength=None,symbol_set=None,verbose=False,parameters=False):
+    r"""
+    Checks if the input is a covering array with given strength.
```

Comment:
Checks -> Check (see [developer guide](https://doc.sagemath.org/html/en/developer/coding_basics.html#the-docstring-of-a-function-content))